Userland/Libraries/LibWeb/HTML/HTMLTextAreaElement.cpp

```diff
@@ -292,7 +292,8 @@ unsigned HTMLTextAreaElement::cols() const
     // The cols and rows attributes are limited to only positive numbers with fallback. The cols IDL attribute's default value is 20.
     if (auto cols_string = get_attribute(HTML::AttributeNames::cols); cols_string.has_value()) {
         if (auto cols = parse_non_negative_integer(*cols_string); cols.has_value())
-            return *cols;
+            if (*cols > 0)
+                return *cols;
```

Comment:
You can update the condition in the `if` statement a line earlier to `cols.has_value() && *cols > 0` for both changes.

This also fixes the issue of missing braces for the middle `if` statement :-)

Reply:
Sure, thanks :)